src/Lykke.HttpClientGenerator/Infrastructure/ExceptionHandlerCallsWrapper.cs

```diff
@@ -26,10 +26,17 @@ public async Task<object> HandleMethodCall(MethodInfo targetMethod, object[] arg
                 var errResponse = ex.GetContentAs<ErrorResponse>();
 
                 if (errResponse != null)
+                {
+                    if (errResponse.ErrorMessage == null)

```

Comment:
Maybe, `string.IsNullOrEmpty` would be better?

Reply:
> Maybe, `string.IsNullOrEmpty` would be better?

I don't think so. Empty string can be applied as a valid part of response. For example when content IS ErrorResponse but with empty ErrorMessage.